rust/src/protocol_types/governance/drep.rs

```diff
@@ -29,6 +30,93 @@ pub enum DRepKind {
     AlwaysNoConfidence,
 }
 
+#[wasm_bindgen]
+#[derive(Clone, Debug, Hash, Eq, Ord, PartialEq, PartialOrd)]
+pub enum DRepExtendedId {
```

Comment:
My suggestion to fully implement CIP-129 GovernanceIdentifier to not have N implementations of different things that just are subset of GovernanceIdentifier